docker/async-slowapp/src/handlers.js

```diff
@@ -273,7 +273,7 @@ async function poll(req, res, next) {
     let poll_count = cookies['poll-count'];
 
     if (poll_count !== undefined) {
-        poll_count = (parseInt(poll_count.split('=')[1]) + 1).toString();
+        poll_counter++
```

Comment:
the cookie was incremented, in order to test that cookies were read a written back correctly ...

Reply:
@vurt007 thanks for the feedback. I've mis understood what this was doing, I'll change it back